core/runtime/runtime_api/impl/runtime_properties_cache_impl.hpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_RUNTIME_RUNTIMEPROPERTIESCACHEIMPL
+#define KAGOME_RUNTIME_RUNTIMEPROPERTIESCACHEIMPL
+
+#include "runtime/runtime_properties_cache.hpp"
+
+namespace kagome::runtime {
+
+  class RuntimePropertiesCacheImpl final : public RuntimePropertiesCache {
+   public:
+    RuntimePropertiesCacheImpl() = default;
+
+    outcome::result<primitives::Version> getVersion(
+        const common::Hash256 &hash,
```

Comment:
it could be either hash of block or hash of state, please give this variable more meaningful name